### PR TITLE
Fix for cookie age unit bug

### PR DESCRIPTION
**Pull request by kyuksel**
- Fixes a bug where cookie expiration time was erroneously converted into milliseconds while being cached in `ProxyService`.

- Factors out helper methods that are now being utilized by multiple `Routes`.

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala [hunk 1]
```diff
@@ -12,15 +12,16 @@ import akka.http.scaladsl.model.headers._
...
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
-trait ProxyRoutes extends UserInfoDirectives with CorsSupport { self: LazyLogging =>
+trait ProxyRoutes extends UserInfoDirectives with CorsSupport with RouteHelper { self: LazyLogging =>
   val proxyService: ProxyService
   implicit val executionContext: ExecutionContext
 
-  protected val tokenCookieName = "LeoToken"
+  private val tokenCookieName = "LeoToken"
```
Comment (rtitle) [inline src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala]: Could we define `tokenCookieName` in `RouteHelper` so it's not duplicated in `LeoRoutes` and `ProxyRoutes`?

I feel like it'd be easy to change it in one place and forget the other.

File: src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala [hunk 2]
```diff
@@ -67,7 +67,7 @@ class ProxyService(proxyConfig: ProxyConfig,
     googleTokenCache.get(token).map {
       case (userInfo, expireTime) =>
         if (expireTime.isAfter(Instant.now))
-          userInfo.copy(tokenExpiresIn = expireTime.toEpochMilli - Instant.now.toEpochMilli)
+          userInfo.copy(tokenExpiresIn = expireTime.getEpochSecond - Instant.now.getEpochSecond)
```
Comment (rtitle) [inline src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala]: 👍

File: src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/RouteHelper.scala [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.dsde.workbench.leonardo.util
+
+import akka.http.scaladsl.model.headers.HttpCookie
+import akka.http.scaladsl.server.Directive0
+import akka.http.scaladsl.server.Directives.setCookie
+import org.broadinstitute.dsde.workbench.model.UserInfo
+
+trait RouteHelper {
```
Comment (rtitle) [inline src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/RouteHelper.scala]: Maybe we could make the trait name more specific, like `CookieHelper`.

**Comment by kyuksel** [comment]
@rtitle Thanks for the feedback! I've addressed them in the penultimate commit. The last commit is to move `BucketHelper` into `util` package, which causes a lot of diff. Sorry about that.

**Pull request merged**